movie/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for movie project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-akq-5q35i!mz-^69j@6ulp2-v_#$*y0a4cb5=*$4crr7si17ca"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
The `DEBUG` setting is set to `True`, which is not recommended for production environments as it can expose sensitive information. Ensure this is set to `False` in production.